src/utilities.js

```diff
@@ -96,12 +96,22 @@ module.exports = class Utilities {
     let statusScreen = await this._testApp.OpenStatusScreen();
     await statusScreen.HeaderSection.Devices.AddCGM();
   }
-  async dismissTidepoolLogin() {
-    try {
-      await match.Label("Welcome_1_Top").swipe('down', 'fast', 0.8);
-    } catch (e) {
-      console.log("onborading is not present on ios 13");
+  async bypassTidepoolOnboarding() {
+    // TODO figure out why hosting views make contained element not visible (and thus cannot tap)
+    // step through the onboarding screens
+    for (var pageCount = 0; pageCount < 7; pageCount++) {
```

Comment:
How long does this skipping take in the overall tests? If all of this is too long, we can add another way to skip everything from the first Welcome page. Perhaps not even a long press.

Reply:
in the order of seconds and under 10 seconds, where most of it is the 4 second long press.